src/agent0/core/hyperdrive/interactive/local_hyperdrive.py

```diff
@@ -340,7 +348,7 @@ def _create_checkpoint(
 
         try:
             tx_receipt = self.interface.create_checkpoint(
-                self._deployed_hyperdrive.deploy_account, checkpoint_time=checkpoint_time
+                self.chain.get_deployer_account(), checkpoint_time=checkpoint_time
```

Comment:
This could potentially be a different deployer account per pool, and is fixed once the pool is deployed. So I think the chain should pass it as an argument on instantiation and then the pool can just own the address as a member variable.

Reply:
Will solve in future PR:
https://github.com/delvtech/agent0/issues/1492